### ISSUE TITLE
SynbioHub down affecting other instances

### ISSUE THREAD

**Issue by fxbuson** [SynBioHub/synbiohub#1553]
After we managed to get our instance of synbiohub online at hub.openbioeconomy.org, everything worked perfectly fine. Yesterday I tried accessing a collection in our instance and it ended up loading endlessly. I found this to be strange, because nothing changed on our server and there should be no traffic at all on that instance.

I also noticed the main synbiohub website is down. Is that why I can't access collections on our instance? My understanding was that instances were completely standalone and wouldn't depend on other groups keeping their servers on.

**Comment by cjmyers** [comment]
You are correct.  You should not have issues based on another instance being down.  Is this collection public or private?  If private, could you send a share link via email?

**Comment by fxbuson** [comment]
The collection was private. I can still send a share link if you want, but I did regain access to it now that synbiohub is back up. No one did anything on our end, as far as I know.

I imagine this is an issue to continue inspecting, if that's not intended behaviour.

**Comment by cjmyers** [comment]
My guess is perhaps virtuoso was down temporarily

**Comment by fxbuson** [comment]
It happened again, but now I suspected it might be an issue with the plugins (since I'm pulling plugins from synbiohub). I checked and the accurat plugins were responding while regular seqviz was not.

Just as I imagined, removing SeqViz (https://seqviz.synbiohub.org/) from my plugins list made the collections open again. Might be something to look into since you'd like it that one plugin you're pulling from another instance malfunctioning wouldn't affect the whole website.

**Comment by cjmyers** [comment]
@zane-perry Can you comment on this issue?

**Comment by cjmyers** [comment]
@zane-perry
